### PR TITLE
[DBAAS-980] enforce use of properly versioned sdk and opm binaries

### PR DESCRIPTION
**Pull request by tchughesiv**
## Description
enforce use of properly versioned sdk and opm binaries

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -272,37 +274,29 @@ bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
 
 .PHONY: opm
-OPM = ./bin/opm
+OPM = ./bin/opm.$(OPM_VERSION)
```
Comment (jianrongzhang89) [inline Makefile:277]: Why this change is needed - to avoid using an outdated opm under the bin folder? It is downloaded during the build.

Comment (tchughesiv) [reply Makefile:277]: Yes, that is one reason for the change... but also to avoid using the wrong version in a user's workstation PATH. Note the other changes made to this make command. With this change, we'd ONLY use the properly versioned `opm` within the repo's relative ./bin dir.

**Review by jeremyary — APPROVED** [review]
